packages/jazz-browser/src/createWebSocketPeerWithReconnection.ts

```diff
@@ -0,0 +1,67 @@
+import { Peer } from "cojson";
+import { createWebSocketPeer } from "cojson-transport-ws";
+
+export function createWebSocketPeerWithReconnection(peer: string, reconnectionTimeout: number | undefined, addPeer: (peer: Peer) => void) {
+    const firstWsPeer = createWebSocketPeer({
+        websocket: new WebSocket(peer),
+        id: peer,
```

Comment:
Changed the id from `options.peer + "@" + new Date().toISOString()` to have a stable identifier